test/acceptance/tests/fixtures/bases/crds/serviceintentions.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: consul.hashicorp.com/v1alpha1
+kind: ServiceIntentions
+metadata:
+  name: intentions
+spec:
+  destination:
+    name: svc1
+  sources:
+    - name: svc2
+      action: allow
```

Comment:
note: we don't need to set `sources[].namespace` because if this is empty, consul will set it to the value of `destination.namespace` (which we're setting through kustomize)